CHANGES.md

```diff
@@ -1,3 +1,42 @@
+# v0.23.0
+
+New Features:
+
+* Added [CloudStack](http://cloudstack.apache.org) provider support (thanks @syed; GH-558).
+* **❢ BREAKING CHANGE:** Added a more flexible benchmark configuration system
```

Comment:
The breaking change is that the old configuration system (which was undocumented, and to my knowledge unused) was removed.

I'll update this message.